eloquentjs/2_2_fizzbuzz.js

```diff
@@ -0,0 +1,10 @@
+function FizzBuzz(iter){
+    for(let i = 1; i <= iter; i++){
+        let output = '';
+        if(i%3==0) output+="Fizz";
```

Comment:
1) You should never use control structures (like for, while, and if) without curly braces. People forgetting they're there and messing up the logic without realizing it is a major failure mode in long lived software systems.
2) In JS you should always use `===` instead of `==` and `!==` over `!=` because of some weird casting "features" from the 90s that can't be removed for backward compatibility reasons.